x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/product_selector/product_selector.test.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import '../../../__mocks__/shallow_usecontext.mock';
+
+import React, { useContext } from 'react';
+import { shallow } from 'enzyme';
+import { EuiPage } from '@elastic/eui';
+
+import '../../../__mocks__/kea.mock';
+import { useValues } from 'kea';
+
+import { ProductSelector } from './';
+import { ProductCard } from '../product_card';
+
+describe('ProductSelector', () => {
+  beforeEach(() => {
+    (useValues as jest.Mock).mockReturnValue({ errorConnecting: false });
+  });
+
+  it('renders the overview page and product cards with no host set', () => {
+    (useContext as jest.Mock).mockImplementationOnce(() => ({ config: { host: '' } }));
+    const wrapper = shallow(<ProductSelector access={{}} />);
+
+    expect(wrapper.find(EuiPage).hasClass('enterpriseSearchOverview')).toBe(true);
+    expect(wrapper.find(ProductCard)).toHaveLength(2);
+  });
+
+  describe('access checks when host is set', () => {
+    beforeEach(() => {
+      (useContext as jest.Mock).mockImplementationOnce(() => ({ config: { host: 'localhost' } }));
+    });
+    it('does not render the App Search card if the user does not have access to AS', () => {
```

Comment:
v small newline nit - tests look great!
```suggestion
    });

    it('does not render the App Search card if the user does not have access to AS', () => {
```